lib/white_bread/suite_configuration.ex

```diff
@@ -0,0 +1,68 @@
+defmodule WhiteBread.SuiteConfiguration do
+  alias WhiteBread.Suite.DuplicateSuiteError
+
+  defmacro __using__(_opts) do
+    quote do
+      import WhiteBread.SuiteConfiguration
+
+      @suites []
+      @before_compile WhiteBread.SuiteConfiguration
+    end
+  end
+
+  defmacro __before_compile__(_env) do
+    quote do
+      def suites do
+        @suites
+          |> throw_if_not_unique
```

Comment:
Just wondering why do you reverse the suites?

I think it should call `unique!(@suites)` which could return nothing or raise error. Then, you could pipe @suites into `Enum.reverse`. Alternatively, you could implement `unique` that returns a tuple containing error information, pattern matching on the resultant message. (I believe this is the recommended approach when error handling in Elixir)

```
def suites do
  unique!(@suites)

  @suites |> Enum.reverse
end
```